### PR TITLE
add calibrate resource error and metrics description for 7.2 and master

### PR DESCRIPTION
**Pull request by CabinfeverB**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-calibrate-resource.md [hunk 1]
```diff
@@ -87,7 +91,14 @@ CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '9m';
...
 
-当时间窗口范围内的负载过低，会导致报错提醒。
+估算容量功能需要监控指标数据，包括 `tikv_cpu_quota`、`tidb_server_maxprocs`、`resource_manager_resource_unit`、`process_cpu_usage`。如果 CPU quota 监控数据为空，会有对应监控项名称的报错，如下面例子所示。
+
+```sql
+CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '60m';
+Error 1105 (HY000): There is no CPU quota metrics, metrics 'tikv_cpu_quota' is empty
+```
+
+当时间窗口范围内的负载过低或者 `resource_manager_resource_unit` 及 `process_cpu_usage` 监控数据缺失，会导致报错提醒。此外，由于 TiKV 未在 macOS 上监控 CPU 使用率，也会导致报错提醒。
```
Comment (hfxsd) [inline sql-statements/sql-statement-calibrate-resource.md:101]:
```suggestion
当时间窗口范围内的负载过低或者 `resource_manager_resource_unit` 及 `process_cpu_usage` 监控数据缺失，会导致报错。此外，由于 TiKV 未在 macOS 上监控 CPU 使用率，也会导致报错。
```

**Review by hfxsd — APPROVED** [review]

File: sql-statements/sql-statement-calibrate-resource.md [hunk 2]
```diff
@@ -87,7 +91,14 @@ CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '9m';
 ERROR 1105 (HY000): the duration of calibration is too short, which could lead to inaccurate output. Please make the duration between 10m0s and 24h0m0s
 ```
 
-当时间窗口范围内的负载过低，会导致报错提醒。
+估算容量功能需要监控指标数据，包括 `tikv_cpu_quota`、`tidb_server_maxprocs`、`resource_manager_resource_unit`、`process_cpu_usage`。如果 CPU quota 监控数据为空，会有对应监控项名称的报错，如下面例子所示。
+
+```sql
+CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '60m';
```
Comment (HuSharp) [inline sql-statements/sql-statement-calibrate-resource.md:97]: 看上去这些 sql 好像中间都缺少了空格

**Review by HuSharp — APPROVED** [review]
rest LGTM!

File: sql-statements/sql-statement-calibrate-resource.md (hunk 2 above)
Comment (CabinfeverB) [reply sql-statements/sql-statement-calibrate-resource.md:97]: 哪里呀

Comment (HuSharp) [reply sql-statements/sql-statement-calibrate-resource.md:97]: 看错了。。忽略这个 comment 把。。😭

File: dashboard/dashboard-resource-manager.md [hunk 3]
```diff
@@ -57,7 +57,9 @@ summary: 介绍如何使用 TiDB Dashboard 的资源管控页面查看资源管
 
     - 如果时间窗口范围不满足 10 分钟至 24 小时的条件，会报错 `Error 1105 (HY000): the duration of calibration is too short, which could lead to inaccurate output. Please make the duration between 10m0s and 24h0m0s`。
 
-    - 如果时间窗口范围内的负载过低，会报错 `Error 1105 (HY000): The workload in selected time window is too low, with which TiDB is unable to reach a capacity estimation; please select another time window with higher workload, or calibrate resource by hardware instead`。
+    - 根据实际负载估算容量功能需要监控指标数据，包括 `tikv_cpu_quota`、`tidb_server_maxprocs`、`resource_manager_resource_unit`、`process_cpu_usage`。如果 CPU quota 监控数据为空，会有对应监控项名称的报错，如 `Error 1105 (HY000): There is no CPU quota metrics, metrics 'tikv_cpu_quota' is empty`。
```
Comment (qiancai) [inline dashboard/dashboard-resource-manager.md:60]:
```suggestion
    - [根据实际负载估算容量](/sql-statements/sql-statement-calibrate-resource.md#根据实际负载估算容量)功能的监控指标包括 `tikv_cpu_quota`、`tidb_server_maxprocs`、`resource_manager_resource_unit`、`process_cpu_usage`。如果 CPU quota 监控数据为空，会有对应监控项名称的报错，如 `Error 1105 (HY000): There is no CPU quota metrics, metrics 'tikv_cpu_quota' is empty`。
```

File: grafana-resource-control-dashboard.md [hunk 4]
```diff
@@ -23,6 +23,8 @@ TiDB 使用[令牌桶算法](https://en.wikipedia.org/wiki/Token_bucket) 做流
 - RRU Per Query：平均每个 SQL 语句消耗的 Read Request Unit 数量。计算方法是将前述 Read Request Unit 监控指标除以当前每秒执行的 SQL 语句数量。
 - WRU：以 Resource Group 为单位进行实时统计的写请求 Write Request Unit 消耗信息。`total` 为当前所有 Resource Group 消耗的 Write Request Unit 之和。
 - WRU Per Query：平均每个 SQL 语句消耗的 Write Request Unit 数量。计算方法是将前述 Write Request Unit 监控指标除以当前每秒执行的 SQL 语句数量。
+- Available RU：以 Resource Group 为单位显示 RU 令牌桶内可用的 token。当指标为 0 时，该 Resource Group 以 RU_PER_SEC 的速度消耗 token，可以认为处于限速状态。
```
Comment (qiancai) [inline grafana-resource-control-dashboard.md:26]:
```suggestion
- Available RU：以 Resource Group 为单位显示 RU 令牌桶内可用的 token。当指标为 0 时，该 Resource Group 将以 `RU_PER_SEC` 指定的速度消耗 token，可以认为处于限速状态。
```

File: sql-statements/sql-statement-calibrate-resource.md [hunk 5]
```diff
@@ -87,7 +91,14 @@ CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '9m';
 ERROR 1105 (HY000): the duration of calibration is too short, which could lead to inaccurate output. Please make the duration between 10m0s and 24h0m0s
 ```
 
-当时间窗口范围内的负载过低，会导致报错提醒。
+估算容量功能需要监控指标数据，包括 `tikv_cpu_quota`、`tidb_server_maxprocs`、`resource_manager_resource_unit`、`process_cpu_usage`。如果 CPU quota 监控数据为空，会有对应监控项名称的报错，如下面例子所示。
```
Comment (qiancai) [inline sql-statements/sql-statement-calibrate-resource.md:94]:
```suggestion
估算容量功能的监控指标包括 `tikv_cpu_quota`、`tidb_server_maxprocs`、`resource_manager_resource_unit`、`process_cpu_usage`。如果 CPU quota 监控数据为空，会有对应监控项名称的报错，如下面例子所示。
```

File: sql-statements/sql-statement-calibrate-resource.md [hunk 6]
```diff
@@ -87,7 +91,14 @@ CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '9m';
 ERROR 1105 (HY000): the duration of calibration is too short, which could lead to inaccurate output. Please make the duration between 10m0s and 24h0m0s
 ```
 
-当时间窗口范围内的负载过低，会导致报错提醒。
+估算容量功能的监控指标包括 `tikv_cpu_quota`、`tidb_server_maxprocs`、`resource_manager_resource_unit`、`process_cpu_usage`。如果 CPU quota 监控数据为空，会有对应监控项名称的报错，如下面例子所示。
```
Comment (qiancai) [inline sql-statements/sql-statement-calibrate-resource.md:94]:
```suggestion
[根据实际负载估算容量](#根据实际负载估算容量)功能的监控指标包括 `tikv_cpu_quota`、`tidb_server_maxprocs`、`resource_manager_resource_unit`、`process_cpu_usage`。如果 CPU quota 监控数据为空，会有对应监控项名称的报错，如下面例子所示。
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**